specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -312,6 +312,145 @@
         }
       }
     },
+    "/custom/models/{modelId}/copy": {
+      "post": {
+        "summary": "Copy Custom Model",
+        "description": "Copy custom model to user specified target Form Recognizer resource.",
```

Comment:
It would be good to have this description give greater clarity about the operation.

Please include:
- custom model stored in this resource (the source)

Reply:
Fixed.